src/iotile/cloud.service.ts

```diff
@@ -165,6 +175,18 @@ export class CloudService {
       });
   }
 
+  public postOrg(org: Org): Observable<Org>  {
+    // return an observable
+    let orgSlug: string = org.slug;
```

Comment:
This is not correct. On a post, there is no slug yet, so you need to post to `/org/`.  Only the returned object will have a `slug` which is computed from the `name`.

Also note that as written, the caller will pass an Org, but you will return a different Org, which may not be what the caller expects. Not technically incorrect, but something to be aware of. This way is simpler, so we can try it this way first.